product_multi_image/__openerp__.py

```diff
@@ -5,13 +5,13 @@
 
 {
     "name": "Multiple Images in Products",
-    "version": "8.0.2.0.0",
-    "author": "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
```

Comment:
@pedrobaeza - I took the liberty of removing your old company and changing the URI to Tecnativa. Let me know if I was incorrect with these changes